pandas/core/generic.py

```diff
@@ -7660,6 +7618,204 @@ def nanptp(values, axis=0, skipna=True):
             ``numpy.ndarray`` method ``ptp``.""",
             nanptp)
 
+    _shared_docs['transform'] = ("""
+    Call function on a Series resp. DataFrame
```

Comment:
summary needs to be a single line, not sure what 'resp.' is.

Reply:
you can add more in the Extended Summary

Reply:
Ok. Summary reduced to one line.  
'resp.' means respectively - but removed to be more compact.  
By 'extended summary' you meant 'Notes' ?

Reply:
Added Notes for clarification